Re-exporting a type when 'verbatimModuleSyntax' is enabled requires using 'export type'

**Describe the bug**
When using tiptap-render-view the [``nuxi typecheck``](https://nuxt.com/docs/api/commands/typecheck) command fails 


```
node_modules/.pnpm/tiptap-render-view@1.0.2_@tiptap+pm@2.11.5/node_modules/tiptap-render-view/types.ts:2:10 - error TS1205: Re-exporting a type when 'verbatimModuleSyntax' is enabled requires using 'export type'.

2 export { JSONContent, Extensions };
           ~~~~~~~~~~~

node_modules/.pnpm/tiptap-render-view@1.0.2_@tiptap+pm@2.11.5/node_modules/tiptap-render-view/types.ts:2:23 - error TS1205: Re-exporting a type when 'verbatimModuleSyntax' is enabled requires using 'export type'.

2 export { JSONContent, Extensions };
                        ~~~~~~~~~~

node_modules/.pnpm/tiptap-render-view@1.0.2_@tiptap+pm@2.11.5/node_modules/tiptap-render-view/vue/types.ts:6:10 - error TS1205: Re-exporting a type when 'verbatimModuleSyntax' is enabled requires using 'export type'.

6 export { Component, VNode };
           ~~~~~~~~~

node_modules/.pnpm/tiptap-render-view@1.0.2_@tiptap+pm@2.11.5/node_modules/tiptap-render-view/vue/types.ts:6:21 - error TS1205: Re-exporting a type when 'verbatimModuleSyntax' is enabled requires using 'export type'.

6 export { Component, VNode };
                      ~~~~~


Found 4 errors in 2 files.
```

**To Reproduce**
Install ``"directus-extension-flexible-editor`` in a new Nuxt App and run npmx nuxi typecheck

**Package Version:**
directus-extension-flexible-editor: 1.8.3
tiptap-render-view 1.0.2

--- a/types.ts
+++ b/types.ts
@@ -1,5 +1,5 @@
 import type { JSONContent, Extensions, AnyExtension } from "@tiptap/core";
-export { JSONContent, Extensions };
+export type { JSONContent, Extensions };
 
 export type RenderedNode<T> = T | string | (T | string)[];
 

--- a/vue/types.ts
+++ b/vue/types.ts
@@ -3,6 +3,6 @@ export type * from "../types";
 import type { ComponentSerializers } from "../types";
 
 import type { VNode, Component } from "vue";
-export { Component, VNode };
+export type { Component, VNode };
 
 export type VueComponentSerializers = ComponentSerializers<Component>;

